test/s3select_perf_test.cpp

```diff
@@ -0,0 +1,488 @@
+  std::stringstream ss;
+  for (auto i = 0U; i < size; ++i) {
+    ss << "2007T,yyyyMMdd-H:m:s" << std::endl;
+    ss << "2019-10-25T01:50:22+04:30,yyyyMMdd-H:m:S" << std::endl;
+    ss << "2008-08-08T15:10-00:00,yyyMd-H:m:s qw" << std::endl;
+    ss << "2021-03-14T03:35:01-01:45,yydaMMMM h m s.n" << std::endl;
+    ss << "2011-11-11T11:11:11Z,yyyyMMdd-H:m:s" << std::endl;
+    ss << "2020-06-30T19:17Z,yydaMMMM h m s.n" << std::endl;
+    ss << "2005-01-15T20:00:00.64+03:00,yyyyMMdd-H:m:s" << std::endl;
+    ss << "2001-02-10T00:12Z,yyyMd-H:m:s qw" << std::endl;
+    ss << "2010-03-10T,yyyyMMdd-H:m:S" << std::endl;
+    ss << "2001-04-19T08:00:00+05:30,yydaMMMM h m s.n" << std::endl;
+  }
+  out = ss.str();
+}
+
+void generate_data_csv(std::string& out, size_t size) {
+  std::stringstream ss;
+  for (auto i = 0U; i < size; ++i) {
+    ss << "1,42926,7334,5.5,Brandise,Letsou,Brandise.Letsou@yopmail.com,worker,2020-10-26T11:21:30.397Z,__" << std::endl;
+    ss << "2,21169,3648,9.0,Zaria,Weinreb,Zaria.Weinreb@yopmail.com,worker,2009-12-02T01:22:45.8327+09:45,__" << std::endl;
+    ss << "3,35581,9091,2.1,Bibby,Primalia,Bibby.Primalia@yopmail.com,doctor,2001-02-27T23:18:23.446633-12:00,__" << std::endl;
+    ss << "4,38388,7345,4.7,Damaris,Arley,Damaris.Arley@yopmail.com,firefighter,1995-08-24T01:40:00+12:30,__" << std::endl;
+    ss << "5,42802,6464,7.0,Georgina,Georas,Georgina.Georas@yopmail.com,worker,2013-01-30T05:27:59.2Z,__" << std::endl;
+    ss << "6,45582,5863,0.1,Kelly,Hamil,Kelly.Hamil@yopmail.com,police officer,1998-03-31T17:25-01:05,__" << std::endl;
+    ss << "7,8548,7665,3.6,Claresta,Flita,Claresta.Flita@yopmail.com,doctor,2007-10-10T22:00:30Z,__" << std::endl;
+    ss << "8,22633,528,5.3,Bibby,Virgin,Bibby.Virgin@yopmail.com,developer,2020-06-30T11:07:01.23323-00:30,__" << std::endl;
+    ss << "9,38439,5645,2.8,Mahalia,Aldric,Mahalia.Aldric@yopmail.com,doctor,2019-04-20T20:21:22.23+05:15,__" << std::endl;
+    ss << "10,6611,7287,1.0,Pamella,Sibyls,Pamella.Sibyls@yopmail.com,police officer,2000-09-13T14:41Z,1_" << std::endl;
+  }
+  out = ss.str();
+}
+
+void generate_single_large_col_csv(std::string& out, size_t size) {
+  std::stringstream ss;
+  for (auto i = 0U; i < size; ++i) {
+    ss << "1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq" << std::endl;
```

Comment:
use std::string( int , char ) instead 
for printing multiple character

Reply:
Done